packages/ProgressBar/src/progress-bar.scss

```diff
@@ -1,60 +1,60 @@
-@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
-@use '~@igloo-ui/tokens/dist/fonts';
+@use "~@igloo-ui/tokens/dist/base10/variables" as tokens;
+@use "~@igloo-ui/tokens/dist/fonts";
+
+@import url('~@hopper-ui/tokens/fonts.css');
+@import url('~@hopper-ui/tokens/tokens.css');
 
 :root {
-  /* Default */
-  --ids-progress-bar-height: #{tokens.$space-2};
-  --ids-progress-bar-radius: #{tokens.$space-1};
-  --ids-progress-bar-background: #{tokens.$grey-200};
-  --ids-progress-bar-background-progress: #{tokens.$electric-blue-500};
-  --ids-progress-bar-background-completed: #{tokens.$seaweed-500};
-
-  /* Compact */
-  --ids-progress-bar-height-compact: #{tokens.$space-1};
-  --ids-progress-bar-radius-compact: calc(#{tokens.$space-1} / 2);
-
-  /* Low contrast */
-  --ids-progress-bar-background-progress-low-contrast: #{tokens.$electric-blue-300};
-  --ids-progress-bar-background-completed-low-contrast: #{tokens.$seaweed-300};
+    /* Default */
+    --ids-progress-bar-height: #{tokens.$space-2};
+    --ids-progress-bar-radius: #{tokens.$space-1};
+    --ids-progress-bar-background: #{tokens.$grey-200};
+    --ids-progress-bar-background-progress: #{tokens.$electric-blue-500};
+    --ids-progress-bar-background-completed: #{tokens.$seaweed-500};
+
+    /* Compact */
+    --ids-progress-bar-height-compact: #{tokens.$space-1};
+    --ids-progress-bar-radius-compact: calc(#{tokens.$space-1} / 2);
+}
+
+[data-brand="workleap"] {
+    /* Default */
+    --ids-progress-bar-height: 0.5rem;
+    --ids-progress-bar-radius: var(--hop-shape-rounded-sm);
```

Comment:
The Figma file uses a pill token, not sure it makes a difference at this size though.